scaling/src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+quarkus.application.name=test-scaling
+quarkus.http.root-path=/api
+quarkus.kubernetes.deployment-target=openshift
```

Comment:
All 3 items above are useless IMHO.

Reply:
Actually I see you depend on the app name, so 2 items above :-)

Reply:
Fixed too